packages/nocodb/src/lib/db/sql-data-mapper/lib/sql/functionMappings/mssql.ts

```diff
@@ -110,6 +110,13 @@ const mssql = {
       END${colAlias}`
     );
   },
+  DATETIME_DIFF: ({ fn, knex, pt, colAlias }: MapFnArgs) => {
+    const date1 = fn(pt.arguments[0]);
+    const date2 = fn(pt.arguments[1]);
+    const unit = fn(pt.arguments[2]);
```

Comment:
since `unit` is optional, feed a default value.

Reply:
I wasn't aware that the unit would be optional. I'll make that change and default it to "minute". If another default makes more sense to you please let me know.

Reply:
Sorry. The logic should be

> If no unit is specified, or the specifier is not recognized by the function, then the output will default to seconds. If the unit specified is not recognized, the output will default to milliseconds.